WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -781,6 +782,19 @@ - (void)showFailedMediaBeforeEditAlert
     [self presentViewController:alertController animated:YES completion:nil];
 }
 
+#pragma mark - Toolbar options
+
+- (void)sharePost
+{
+    if ([self.post isKindOfClass:[Post class]]) {
```

Comment:
Again, super nitpicky,  but maybe return if its not a post class and move the work out of the conditional?